pkg/protocol/protocol_fork.go

```diff
@@ -196,7 +199,7 @@ func (p *Protocol) processFork(fork *chainmanager.Fork) (anchorBlockIDs iotago.B
 	ch := make(chan *commitmentVerificationResult)
 	defer close(ch)
 
-	commitmentVerifier := NewCommitmentVerifier(p.MainEngineInstance(), fork.ForkingPoint)
+	commitmentVerifier := NewCommitmentVerifier(p.MainEngineInstance(), fork.MainChain.Commitment(fork.ForkingPoint.Index()-1).Commitment())
```

Comment:
Isn't the ForkingPoint the last common commitment between chains already? Why you pass the -1?

Reply:
Also, the `forkingPoint` that you save in the `CommitmentVerifier` is never used: only its CW is saved during initialization.

Reply:
Forking point is the first different commitment. We can maybe just pass the weight of the previous commitment. It is needed to correctly compute the cumulative weight.